pinot-common/src/main/java/org/apache/pinot/common/config/IndexingConfig.java

```diff
@@ -87,6 +87,9 @@
   @ConfigKey("aggregateMetrics")
   private boolean _aggregateMetrics;
 
+  @ConfigKey("nullHandlingEnabled")
```

Comment:
`allowNullValue` is more clear?

Reply:
I'll keep this unless there's a strong objection.